cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -468,6 +468,7 @@ private class ServiceImpl(
 
         networkClient.storeCcpaChoice(postConsentParams)
             .executeOnRight { postConsentResponse ->
+                campaignManager.ccpaUuid = postConsentResponse.uuid
                 campaignManager.ccpaConsentStatus?.uuid = postConsentResponse.uuid
```

Comment:
This assignment doesn’t have any effect since on line 473 we override the `ccpaConsentStatus`, we can remove this line